core/version/src/lib.rs

```diff
@@ -1,19 +1,20 @@
 #[macro_use]
 extern crate diesel;
+#[macro_use]
+extern crate diesel_migrations;
 
 pub(crate) mod db;
 mod rest;
-
-pub use rest::VersionService;
+pub mod service;
 
 pub mod notifier {
+    use metrics::counter;
     use std::time::Duration;
-    use tokio::time;
 
     use ya_persistence::executor::DbExecutor;
 
-    const UPDATE_CURL: &'static str = "curl -sSf https://join.golem.network/as-provider | bash -";
-    const SILENCE_CMD: &'static str = "yagna update skip";
```

Comment:
Are those wrong?

Reply:
sorry. Removed it by misunderstanding. Reverted and fixed skip command as it is `yagna version skip`